config/locales/fr.yml

```diff
@@ -167,17 +167,17 @@ fr:
   faker:
     address:
       country_code:
-        - FR
```

Comment:
I hope these unindented lines FRXX do not effect behavior of the code, seems strange that the dashes are on the same level as the parent :thinking: